apps/condo/domains/ticket/components/TicketsCounter/UnreadTicketsCounterContext.tsx

```diff
@@ -0,0 +1,80 @@
+import { createContext, useCallback, useContext, useEffect, useRef, useState } from 'react'
+import { useLazyQuery, useMutation } from '@apollo/client'
+import { useAuth } from '@core/next/auth'
+import { GET_TICKET_LAST_TIME_VIEWED_QUERY, SET_TICKET_LAST_TIME_VIEWED_MUTATION } from '@condo/domains/ticket/gql'
+import { Ticket } from '@condo/domains/ticket/utils/clientSchema'
+import { SortTicketsBy } from '../../../../schema'
+
+type UnreadTicketsCounterContext = {
+    numberOfUnreadTickets: number
+    syncWithServer: (d: number) => void
+}
+const UnreadTicketsCounterContext = createContext<UnreadTicketsCounterContext>({ 
+    numberOfUnreadTickets: 0,
+    syncWithServer: void 0,
+})
+
+const TICKETS_UPDATE_DELAY = 5000
+
+export const UnreadTicketsFetcher: React.FC = ({ children }) => {
+    const { isAuthenticated } = useAuth()
+    const [numberOfUnreadTickets, setNumberOfUnreadTickets] = useState(0)
+    const lastTimeout = useRef(-1)
+    const lastTimeTicketsViewed = useRef(0)
+    
+    const [getTicketLastTimeViewedFromServer] = useLazyQuery(GET_TICKET_LAST_TIME_VIEWED_QUERY, {
+        onCompleted: ({ result: { syncedTicketLastTimeViewed } }) => {
+            lastTimeTicketsViewed.current = syncedTicketLastTimeViewed || 0
+            pollingFn()
+        },
+    })
+    const [syncTicketLastTimeViewedMutation] = useMutation(SET_TICKET_LAST_TIME_VIEWED_MUTATION)
```

Comment:
Please, use name, consistent to name of mutation.

```suggestion
    const [setTicketLastTimeViewedMutation] = useMutation(SET_TICKET_LAST_TIME_VIEWED_MUTATION)
```